docs/resources/sql_table.md

```diff
@@ -0,0 +1,95 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_sql_table (Resource)
+
+Within a metastore, Unity Catalog provides a 3-level namespace for organizing data: Catalogs, databases (also called schemas), and tables / views.
+
+A `databricks_sql_table` is contained within [databricks_schema](schema.md).
+
+## Example Usage
+
+```hcl
+resource "databricks_catalog" "sandbox" {
+  metastore_id = databricks_metastore.this.id
+  name         = "sandbox"
+  comment      = "this catalog is managed by terraform"
+  properties = {
+    purpose = "testing"
+  }
+}
+
+resource "databricks_schema" "things" {
+  catalog_name = databricks_catalog.sandbox.id
+  name         = "things"
+  comment      = "this database is managed by terraform"
+  properties = {
+    kind = "various"
+  }
+}
+
+resource "databricks_sql_table" "thing" {
+  provider           = databricks.workspace
+  name               = "quickstart_table"
+  catalog_name       = databricks_catalog.sandbox.id
+  schema_name        = databricks_schema.things.id
+  table_type         = "MANAGED"
+  data_source_format = "DELTA"
+  storage_location   = ""
+
+  column {
+    name      = "id"
+    type_text = "int"
+  }
+  column {
+    name      = "name"
+    type_text = "varchar(64)"
+    comment   = "name of thing"
+  }
+  comment = "this table is managed by terraform"
+}
+
+resource "databricks_sql_table" "thing_view" {
+  provider           = databricks.workspace
+  name               = "quickstart_table_view"
+  catalog_name       = databricks_catalog.sandbox.id
+  schema_name        = databricks_schema.things.id
+  table_type         = "VIEW"
+
+  view_definition    = format("SELECT name FROM %s WHERE id == 1", databricks_sql_table.thing.id)
+
+  comment = "this view is managed by terraform"
```

Comment:
please also provide example with setting of `cluster_id` property explicitly - I think implicit cluster has a risk of race conditions :)